deepchem/molnet/load_function/material_datasets/tests/test_load_Pt_NO_surface_adsorbate_energy.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
Currently, we're discussing how to test the molnet functions. So, can you remove this?
Calling `load_Platinum_Adsorption` will block the CI